custom_components/tuya_local/devices/ustellar_ut99911_rgb.yaml

```diff
@@ -65,8 +53,21 @@ entities:
               min: 0
               max: 1000
       - id: 103
-        name: line_sequence_adjustment
-        type: integer
+        type: string
+        name: RGB Order
+        mapping:
+          - dps_val: 1
```

Comment:
Numbers used as a string should have "quotes"

Reply:
Have set `type` back to `integer` as the values are always numbers from this DP. 👍🏼